tests/foreman/cli/test_host.py

```diff
@@ -1,5 +1,8 @@
 """CLI tests for ``hammer host``."""
```

Comment:
I am pretty sure some test names are larger in length to what pylint expects.  In that case implement this to silence it for the module:
https://github.com/SatelliteQE/robottelo/blob/master/tests/foreman/ui/test_ldapauthsource.py#L1